cpp/csp/python/cspbaselibimpl.cpp

```diff
@@ -6,6 +7,7 @@
 
 static void * init_nparray()
 {
+    csp::python::AcquireGIL gil;
```

Comment:
On the surface it looks like we acquire the GIL but then never release it (however a look at the AcquireGIL code shows that we release when the object is destroyed i.e. we go out of scope).

A better name would be `AcquireGILInScope` or something.